src/analyze/analyze.c

```diff
@@ -66,10 +68,13 @@
 static enum dot {
```

Comment:
we should probably give this a better name now. maybe:

```c
typedef enum DotDependencyKind {
        DOT_DEPENDENCY_ALL,
        DOT_DEPENDENCY_ORDER,
…
        _DOT_DEPENDENCY_KIND_MAX,
        _DOT_DEPENDENCY_KIND_INVALID = -1,
} DotDependencyKind;

static enum DotDependencyKind arg_dot_dependency_kind = DOT_DEPENDENCY_ALL;
```

Reply:
done